index.js

```diff
@@ -49,38 +52,34 @@ var cli = new Liftoff({
   ],
 });
 
-var usage =
-  '\n' + chalk.bold('Usage:') +
-  ' gulp ' + chalk.blue('[options]') + ' tasks';
-
 var parser = yargs
   .help(false)
   .version(false)
   .detectLocale(false)
-  .usage(usage)
+  .showHelpOnFail(false)
+  .exitProcess(false)
+  .fail(onFail)
   .options(cliOptions);
 
 var opts = parser.parse();
 
 // Set up event listeners for logging temporarily.
-toConsole(log, opts);
+// TODO: Rework console logging before we can set up proper config
+// Possibly by batching messages in gulplog until listeners are attached
+var cleanupListeners = toConsole(log, opts, buildTranslations());
```

Comment:
This is a general bug with our libraries. I'll create a "known issue" on gulp-cli for us to fix this in our next breaking version.

Reply:
This could be a temporary solution, but I want people to be able to use typescript, babel, and other loaders so I think we'll just create the known issue.

I'm already planning the changes to Liftoff to support this, but we need top-level await (in gulp 6).

Reply:
#264